helm/hpcc/values.schema.json

```diff
@@ -799,70 +798,81 @@
       }
     },
     "sasha-coalescer": {
-      "type": "object",
-      "allOf": [{ "$ref": "#/definitions/sashacommon" }],
-      "properties": {
-        "service": { "const": "coalescer" },
-        "minDeltaSize": {
-          "type": "integer",
-          "description": "Coalescing will only begin, if the delta size is above this threashold (K)"
+      "oneOf": [
+        {
+          "type": "object",
+          "allOf": [{ "$ref": "#/definitions/sashacommon" }],
+          "properties": {
+            "minDeltaSize": {
+              "type": "integer",
+              "description": "Coalescing will only begin, if the delta size is above this threashold (K)"
+            },
+            "disabled": {},
+            "interval": {},
+            "servicePort": {},
+            "port": {},
+            "at" : {},
+            "throttle": {},
+            "storage": {},
+            "resources": {}
```

Comment:
The attributes above do not have a type or description

Reply:
A lot of fields here.  Could we simplify values.yaml by defining defaults for most of these?

Reply:
actually, they are still needed.
The schemas are defined and inherited via allOf and "sashacommon".
They are necessary to define here too, because the additionalProperties: false only applies to "properties", so although the schema are inherited, they need defining here to avoid additionalProperties rejecting them when they appear in values.yaml.
For reference see: https://json-schema.org/understanding-json-schema/reference/combining.html.

However, I did notice a mistake in the use of allOf, I'll change in coming commit